docs/configuration.rst

```diff
@@ -69,6 +69,8 @@ The ``MESSAGE_SIZE_LIMIT`` is the maximum size of a single email. It should not
 be too low to avoid dropping legitimate emails and should not be too high to
 avoid filling the disks with large junk emails.
 
+.. note:: Albeit ``MESSAGE_SIZE_LIMIT=0`` (no limit) is possible technically, this likely won't be the case for your external recipients. Use with care.
```

Comment:
I'd rather we simplified and documented if necessary that a high limit can be set. Unlimited just doesn't exist in CS.

Reply:
CS? i checked everywhere and didn't see no incompatibilities, but you know better where this message limit size variable is used then i do. It's perfectly fine to not allow size=0 but in this case it has to be documented is all i'm saying.
you can close this PR and simply keep the doc update part by saying 0 is not allowed, it's up to you guys.

Reply:
Computer Science.

I'd rather we documented that we expect a value... or pick a reasonable default (say 10MiB)

Reply:
Well technically it's possible to use 0, as the various software devocot postifx nginx php etc all accept this value as 'unlimited'.
Whether it's okay or not to let users use 0, is beyond me. this is not my project, i just try to make this work with 0 and users do what they want. we can also have a vote if you like? how do we poll this question out?

i vote to let the users decide to use 0 or fixed values :)

Reply:
I tested your updated patch-3 branch.  It fails on a 9MB attachment after doing docker-compose up -d (when recreating the whole infrastructure) but if later I do docker-compose restart webmail then the 9MB attachment succeeds.

Actually, the problem occurs when you do docker-compose up -d EVEN IF ONLY the webmail container is recreated (i.e. it does not require all containers to be recreated as I had initially reported on #2204).  This seems to indicate that when the start.py in webmail runs the changed file created by the in-place sed command is not used.  I will try to redo this without doing a sed in-place edit.

Reply:
I think that the rainloop.conf in http.d does not take effect when doing a docker-compose up -d command.  Will look into that.

Reply:
I have done a ton of testing.  I cannot understand what is going on. I am using a very recent docker snap version:
```
snap list docker
Name    Version   Rev   Tracking       Publisher   Notes
docker  20.10.11  1458  latest/stable  canonical✓  -
```
running on ubuntu 20.04 so I do not believe this to be a docker bug.  @audioscavenger  Eric, are you seeing this? I created a branch test-startup on https://github.com/marioja/Mailu/tree/test-startup which will fail when doing docker-compose up -d and work after a docker-compose restart webmail.  I output into the webmail/out folder a start.log file with information about just before nginx is started in the webmail container and you can see the output from the docker-compose up and docker-compose restart (the output begins with a bunch of ========== everytime the webmail container starts).  I would like to know if I am the only one to see this.

Reply:
I found out why there is a difference in behaviour in webmail between creating the container and restarting it.  If you look at PR #2207 you can read this comment https://github.com/Mailu/Mailu/pull/2207#discussion_r805455813 for details